AnalyticsAdmin/owlbot.py

```diff
@@ -20,18 +20,22 @@
 
 import synthtool as s
 from synthtool.languages import php
+from synthtool import _tracked_paths
 
 logging.basicConfig(level=logging.DEBUG)
 
 src = Path(f"../{php.STAGING_DIR}/AnalyticsAdmin").resolve()
 dest = Path().resolve()
 
+# Added so that we can pass copy_excludes in the owlbot_main() call
+_tracked_paths.add(src)
```

Comment:
This is what hung me up a bit, didn't expect to need to add this in - it was preventing the excludes from working properly. Should be good to go now @bshaffer 👍.